### PR TITLE
Get Number of New Watchlist Instance Endpoint

### PR DESCRIPTION
**Pull request by victorhuangwq**
## Description

This endpoint allows you to get the number of new watchlist instance

## Motivation and Context

This is for the purpose of the notification badge on the course page, as we would prefer not needing to get all watchlist instances just to get the number of new ones.

## How Has This Been Tested?

Started with no new watchlist instances in the records
1. called endpoint, returns `{num_new:0}`
1. Add a new instance, called endpoint, returns `{num_new:1}`
1. Add another new instance, called endpoint, returns `{num_new:2}`
1. Call with invalid course, gets redirected to 404.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

### CODE REVIEW

File: app/controllers/metrics_controller.rb [hunk 1]
```diff
@@ -33,10 +33,25 @@ def get_watchlist_instances
 		# On error, a 404 error is returned
 		begin
 			course_name = params[:course_name]
-			conditions = WatchlistInstance.get_instances_for_course(course_name)
-			render json: conditions
+			instances = WatchlistInstance.get_instances_for_course(course_name)
+			render json: instances
```
Comment (xinyis991105) [inline app/controllers/metrics_controller.rb:37]: To be consistent across all endpoints, should we also add status: :ok here as well? If necessary, I'll make sure I add these for subsequent endpoints too.

**Review by xinyis991105 — CHANGES_REQUESTED** [review]
To be consistent, can we also add :status => :ok

File: app/controllers/metrics_controller.rb (hunk 1 above)
Comment (victorhuangwq) [reply app/controllers/metrics_controller.rb:37]: Yes, I agree that we should return status =>:ok. I will amend that as well

**Review by xinyis991105 — APPROVED** [review]
LGTM!

**Pull request merged**